### ISSUE TITLE
Embed builder class

### ISSUE THREAD

**Issue by javis86** [ToshiroZ/Discord-Webhook#3]
I've been working also with a Class taken from the package Discord.Net which is useful for my library and Discord-Webhook.
Example building and embed:

```cs
private static WebhookObject BuildExceptionObject(LogEvent logEvent, IFormatProvider formatProvider)
        {
            var stackTrace = logEvent.Exception.StackTrace;
            stackTrace = FormatStackTrace(stackTrace);

            var embedBuilder = new EmbedBuilder()
                .WithTitle("Error")
                .WithDescription(logEvent.Exception.Message)
                .WithColor(GetColorFromLevel(logEvent.Level))
                .WithThumbnail("https://raw.githubusercontent.com/javis86/Serilog.Sinks.Discord/master/Resources/error.png")
                .AddField("Type", logEvent.Exception.GetType().Name, true)
                .AddField("TimeStamp", logEvent.Timestamp.ToString(), true)
                .AddField("Message", logEvent.Exception.Message)
                .AddField("StackTrace", stackTrace)
                .AddFieldsRange(GetFieldsFromLogEventProperties(logEvent.Properties));

            var embed = embedBuilder.Build();
            
            var obj = new WebhookObject {embeds = new List<Embed> {embed}};
            return obj;
        }
```

Here is the link to the class: [EmbedBuilder.cs](https://github.com/javis86/Serilog.Sinks.Discord.Lite/blob/f5436d0624a3937b97889cbc6f4b75c1b851dc50/Helpers/EmbedBuilder.cs)

I could add it too.

**Comment by ToshiroZ** [comment]
I can add my own solution to this problem soon. I see reasons to add it as it would make the making of WebhookObjects easier. If you want to make a pull request with it I can accept it no problem.

**Comment by javis86** [comment]
Merged in #4
